cypress.json

```diff
@@ -14,5 +14,6 @@
   "viewportWidth": 1280,
   "viewportHeight": 780,
   "screenshotsFolder": "cypress/snapshots/actual",
-  "trashAssetsBeforeRuns": true
+  "trashAssetsBeforeRuns": true,
+  "projectId": "ihjv34"
```

Comment:
qq: is this required and/or something we can expose publicly?

Reply:
Per Cypress, yes and yes. https://docs.cypress.io/guides/dashboard/projects#Set-up-a-project-to-record The ID is also visible in the dashboard URL.